Description: Upstream changes introduced in version 0.6.0+svn20100107-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 fontmatrix (0.6.0+svn20100107-1) unstable; urgency=low
 .
   * Upstream svn snapshot.
   * debian/source/format: Switched to dpkg-source 3.0 (quilt) format.
   * debian/control: Updated Standards-Version to 3.8.3.
   * debian/patches/01_fminfodisplay.cpp.patch: fixes FTBFS on arm. Thanks to
     Michael Casadevall <mcasadevall@debian.org>.
 .
 The person named in the Author field signed this changelog entry.
Author: Oleksandr Moskalenko <malex@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- fontmatrix-0.6.0+svn20100107.orig/src/fminfodisplay.cpp
+++ fontmatrix-0.6.0+svn20100107/src/fminfodisplay.cpp
@@ -151,7 +151,7 @@ QString FMInfoDisplay::writeSVGPreview(F
 				GlyphToSVGHelper gtsh ( gpi->path(), tf );
 				svg += gtsh.getSVGPath() + "\n";
 				horOffset += gpi->data(GLYPH_DATA_HADVANCE).toDouble() * scaleFactor;
-				maxHeight = qMax<qreal> ( gtsh.getRect().height(), maxHeight );
+				maxHeight = qMax ( gtsh.getRect().height(), maxHeight );
 				tf.translate( gpi->data(GLYPH_DATA_HADVANCE).toDouble()  * scaleFactor,0 );
 				delete gpi;
 			}
